.github/workflows/ci.yml

```diff
@@ -138,7 +138,6 @@ jobs:
             src/types/v3
             src/abis/types
             src/state/data
-            src/locales
```

Comment:
Probably is best if we define a var with these paths so we keep this file DRY-er (easy to miss one file here)